src/citrine/informatics/predictors.py

```diff
@@ -272,3 +276,122 @@ def __str__(self):
     def post_build(self, project_id: UUID, data: dict):
         """Creates the predictor report object."""
         self.report = ReportResource(project_id, self.session).get(data['id'])
+
+
+class MolecularStructureFeaturizer(Serializable['MolecularStructureFeaturizer'], Predictor):
+    """
+    [ALPHA] A "batteries-included" featurizer for organic molecules. Powered by CDK.
+
+    Parameters
+    ----------
+    name: str
+        name of the configuration
+    description: str
+        the description of the predictor
+    descriptor: MolecularStructureDescriptor
+        the descriptor to featurize
+    features: List[str]
+        the list of features to compute. Valid values include:
+        - all
+        - standard
+        - expensive
```

Comment:
I assume the first three are sets of features rather than particular ones; probably worth mentioning or inserting an empty line.